### ISSUE TITLE
Exercise 3 - solution is correct but nothing happens

### ISSUE THREAD

**Issue by oren0e** [dibgerge/ml-coursera-python-assignments#51]
Hi,

On the last task when you have to implement the `predict()` function for neural nets - I test it and get the correct 97.5% training set accuracy but then when I submit using the grader I don't get the points. It just shows " 0 / 30" for the last task without any additional message. Is it a bug or a mistake on my end?
I have tried to generate a new token on Coursera and resubmit but it did not help.

**Comment by oren0e** [comment]
I'm sorry it was a subtlety in my code

**Comment by sabian2008** [comment]
Would you mind sharing what was that subtlety? I'm experiencing the exact same issue.

**Comment by oren0e** [comment]
It has been some time since I've touched this but from what I recall the problem was that I tried to compute two steps in one go. This had nothing to do with the code itself in this repo.
